test/integration/globals/app-nav.js

```diff
@@ -528,13 +539,13 @@ context('App Nav', function() {
         last_active_at: testTs(),
       },
       relationships: {
-        team: { data: { id: '11111' } },
+        team: getRelationship(teamCoordinator),
         workspaces: { data: _.times(10, n => {
```

Comment:
Not sure how to handle this one. I'm having a hard time getting the data formatted correctly using `getWorkspace()`/`getWorkspaces()` 🤔

Reply:
probably just `getRelationships(_.times(10.... return { id: uuid(...) }..), 'workspaces'),`

Reply:
Yup, that works 👍